### ISSUE TITLE
Asterisks not shown for required fields

### ISSUE THREAD

**Issue by tylersand** [azure-ad-b2c/html-templates#15]
Self-asserted page v2.1.18 adds an asterisk for required fields. Using the Azure Blue template as a custom page, these are not shown.

![image](https://user-images.githubusercontent.com/16942321/220223698-c3db5fa9-28f4-4bf5-8d33-35c82fb360af.png)

Expected (as shown in non-custom page):
![image](https://user-images.githubusercontent.com/16942321/220223605-f902a910-fd88-42fe-99d9-42048b9bce58.png)

**Comment by shibayan** [comment]
I encountered the same problem, but was able to fix it by adding the query parameter `pageFlavor=oceanBlue` to the URL specified in `LoadUri`.

**Comment by tylersand** [comment]
> I encountered the same problem, but was able to fix it by adding the query parameter `pageFlavor=oceanBlue` to the URL specified in `LoadUri`.

When you say LoadUri, do you mean just adding it to the query string for the Custom Page URI?
![image](https://user-images.githubusercontent.com/16942321/232088957-967e7402-231f-41e6-869a-22f854ed242d.png)

**Comment by shibayan** [comment]
Yes, that's right, LoadUri was talking about a custom policy, so never mind.

**Comment by tylersand** [comment]
I was able to add this JavaScript to the custom page to get it to work as expected:
$('input,textarea,select').filter('[aria-required]').after('<span class="required">*</span>');

**Comment by sebastianpatten** [comment]
I have the same issue
